content/en/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/ios.md

```diff
@@ -308,11 +353,14 @@ You can use the following properties in `RUM.Configuration` when enabling RUM:
 `longTaskEventMapper`
 : Sets the data scrubbing callback for long tasks. This can be used to modify or drop long task events before they are sent to Datadog. For more information, see [Modify or drop RUM events](#modify-or-drop-rum-events).
 
-`vitalsUpdateFrequency`
-: Sets the preferred frequency of collecting mobile vitals. Available values include: `.frequent` (every 100ms), `.average` (every 500ms), `.rare` (every 1s), and `.never` (disable vitals monitoring).
+`onSessionStart`
+: Enter a method that gets called when RUM starts the session.
 
-`appHangThreshold`
-: Sets the threshold for reporting app hangs. The minimum allowed value for this option is `0.1` seconds. To disable app hangs reporting, set this to `nil`. For more information, see [Add app hang reporting][10].
+`customEndpoint`
+: A custom server URL for sending RUM data.
+
+`telemetrySampleRate`
+: The sampling rate for the SDK internal telemetry utilized by Datadog. This must be a value between `0` and `100`. By default, this is set to `20`.
```

Comment:
I don't understand this as written. Can you elaborate?

Reply:
Added more detail.